src/api/app/views/webui/repositories/_edit_repository.html.haml

```diff
@@ -0,0 +1,43 @@
+.box{ id: "#{valid_xml_id(repository.name)}", style: "margin: 1em" }
+  %h2.box-header
+    Edit #{repository.name}
+  = form_tag({ action: :update }, id: "update_target_form-#{valid_xml_id(repository.name)}") do
+    %p
+      %b Architectures:
+      - Architecture.available.each do |architecture|
+        = check_box_tag "arch[#{architecture}]", "", repository.architectures.include?(architecture), class: 'arch_check_box'
+        = architecture.name
+      = hidden_field_tag :project, project
+      = hidden_field_tag :repo, repository.name
+    %div{ style: "margin-bottom: 1.2em" }
+      %p
+        %b Additional package repositories (searched in descending order):
+      %div{ style: "margin-left: 30px" }
+        - begin
+          - repository.path_elements.each do |path|
+            - if path.link == Repository.deleted_instance
+              = image_tag 'exclamation.png'
+              Target repository has been removed
+            - else
+              %span= path.link.project_name + "/" + path.link.name
+              - if repository.path_elements.size > 1
+                - unless path == repository.path_elements.first
```

Comment:
do you prefer `unless ==` than `if !=`? For me the second is clearer. It seems that this is not included in Ruby style guide. :thinking: